src/router/index.ts

```diff
@@ -16,6 +17,7 @@ const Router = express.Router()
 Router.use('/login', login)
 Router.use('/users', isAuthorized, users)
 Router.use('/status', status)
+Router.use('/user-course', isAuthorized, userCourse)
```

Comment:
Would we want to make this plural? 

```suggestion
Router.use('/user-courses', isAuthorized, userCourse)
```